apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -102,6 +97,9 @@ export const AIAssistant = ({ className }: AIAssistantProps) => {
   const inputRef = useRef<HTMLTextAreaElement>(null)
   const { ref: scrollContainerRef, isSticky, scrollToEnd } = useAutoScroll()
 
+  // TODO: How do we handle IS_PLATFORM here?
+  let aiOptInLevel = useOrgAiOptInLevel()
```

Comment:
wondering what do you mean for this one? maybe this [comment](https://github.com/supabase/supabase/pull/35318/files#r2092759401) might be related?

also use a `const` instead of a `let`